pkg/maintenance/postgresql.go

```diff
@@ -121,10 +122,33 @@ func (p *PostgreSQL) DoMaintenance(ctx context.Context) error {
 		return fmt.Errorf("cannot watch for maintenance sgdbops resources: %v", err)
 	}
 
-	p.log.Info("Repacking databases...")
-	err = p.createRepack(sgCluster.GetName())
+	p.log.Info("Setting vacuum and repack variables")
+	err = p.setVacuumRepack()
 	if err != nil {
-		return fmt.Errorf("cannot create repack: %v", err)
+		return fmt.Errorf("cannot set vacuum and repack variables: %v", err)
+	}
+
+	if p.Vacuum {
+		p.log.Info("Vacuuming databases...")
+		err = p.createVacuum(sgCluster.GetName())
+		if err != nil {
+			return fmt.Errorf("cannot create vacuum: %v", err)
+		}
+	}
+
+	if p.Repack {
+		p.log.Info("Repacking databases...")
+		err = p.createRepack(sgCluster.GetName())
+		if err != nil {
+			return fmt.Errorf("cannot create repack: %v", err)
+		}
+	}
+	// default to repack if for some reason it was possible to disable both vacuum and repack
```

Comment:
We have to add this important information into the description of our repack field in the CRD.

Reply:
fixed